cookbooks/anisotropic_viscosity/av_material.cc

```diff
@@ -0,0 +1,858 @@
+#include <aspect/introspection.h>
+#include <aspect/material_model/interface.h>
+#include <aspect/plugins.h>
+#include <aspect/simulator/assemblers/interface.h>
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/parameter_handler.h>
+#include <deal.II/base/patterns.h>
+#include <deal.II/base/point.h>
+#include <deal.II/base/quadrature.h>
+#include <deal.II/base/table_indices.h>
+#include <deal.II/base/tensor.h>
+#include <deal.II/fe/fe.h>
+#include <deal.II/fe/fe_update_flags.h>
+#include <deal.II/grid/tria_iterator_base.h>
+#include <array>
+#include <cmath>
+#include <functional>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#ifndef __aspect__av_material_h
+#define __aspect__av_material_h
+
+#include <aspect/simulator_access.h>
+#include <aspect/simulator.h>
+#include <aspect/global.h>
+
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/fe/fe_values.h>
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/base/geometry_info.h>
+#include <aspect/simulator_access.h>
+
+#include <aspect/material_model/simple.h>
+#include <aspect/material_model/grain_size.h>
+#include <aspect/heating_model/interface.h>
+#include <aspect/gravity_model/interface.h>
+#include <aspect/simulator/assemblers/stokes.h>
+#include <aspect/simulator_signals.h>
+#include <aspect/postprocess/particles.h>
+
+#include <deal.II/base/symmetric_tensor.h>
+#include <deal.II/base/signaling_nan.h>
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+              /**
+               * Additional output fields for anisotropic viscosities to be added to
+               * the MaterialModel::MaterialModelOutputs structure and filled in the
+               * MaterialModel::Interface::evaluate() function.
+               */
+              template <int dim>
+              class AnisotropicViscosity : public NamedAdditionalMaterialOutputs<dim>

```

Comment:
I think you haven't run the indentation script, or did the script simply not pick up your file?